ParadiseSS13/Paradise
# The degrilling: map windows no longer have grilles under them unless electrified or space adjacent

## What Does This PR Do
This PR removes the grilles from most of the **roundstart windows**. Only the ones that have an actual gameplay difference have been kept, which are those with a cable node under them. These are usually located in high security areas, for safety reasons.

In order to keep mapping changes to a minimum, the base window spawner has been modified to not spawn grilles by default. New `/grilled` subtypes of each window variant have been added, and new `icon_state`s have been made for grille-free windows.

The mapping changes have been done automatically with the help of @warriorstar-orion. And the are paths have been updated following #21681 changes by using the supplied UpdatePaths.

**UPDATE:** following the request, space adjacent windows are back to being grilled. Be aware this has introduced **manual map changes**, so reviewing might take a bit longer than before.

**UPDATE 2:** Went over the `RandomRuins/SpaceRuins` and replaced the spacefaring windows to be grilled as well. Was less effort than I previously thought.

## Why It's Good For The Game
Honestly? :sparkles: *Aesthetics* :sparkles: 

But the other main reason is that, as of right now, all windows have grilles under them. This offers no gameplay difference besides a couple more hits to destroy a window, or no effort if you have wirecutters with you. It only adds visual clutter.

With this change, players can gain a visual representation of what is a **high security area**, on top of new antag players being able to recognize what windows will shock them (bridge, head offices, brig etc.).

On top of that, this is part one of the window changes I've planned. The idea is to **stop using reinforced windows everywhere**, and keep them only for secure areas + space adjacent windows. Most likely, this will include a buff to normal windows, because currently they are weaker than wet paper.

I am aware that the plating under the windows, now that it's nicely visible, might not look the best as it clearly marks the breaks between each window tile instead of giving the illusion of one continuous window. Honestly, each time I see it I have a different opinion. Whether basic flooring is preferred under all windows or not is something that might have to be discussed in this PR's comments. It is also possible to make some flooring decal that would fit the window floors better?

Also, the icon states for the non-grilled reinforced windows might be a bit too similar to the grilled ones. They were made following the style of the existing ones, but I can change them if it is necessary for better mapping.

## Images of changes
Couple of images showing both types:

![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/cbc09ba5-4aa3-4e71-aead-520500a2e3e7)
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/cd19427e-b1f9-4635-a22a-f563281f6442)

Check that all spawners work correctly:

![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/8ec6c618-dde5-4da0-94fd-10733bcfcc65)

Icon states of new non-grilled spawners:
![imagen](https://github.com/ParadiseSS13/Paradise/assets/58085266/ecf73605-6f73-40f3-af87-ff4692a66ceb)


## Testing
- Log into game, create all types of window spawners and ensure they work properly
- Check that most windows do not have grilles
- Ensure that areas such as Bridge, Brig, Heads offices etc. **do** maintain their grilles
- Start making some nice hamb- wait no

## Changelog
:cl: GoR, warriorstar
tweak: Removes grilles from most windows in all maps, keeping them only on electrified ones.
/:cl:
